src/vec.cpp

```diff
@@ -1217,6 +1217,15 @@ int symmetry::multiplicity() const {
     return g;
 }
 
+int symmetry::multiplicity(vec &x) const {
+  int m = multiplicity();
+  for (int n=1; n<m; ++n){
+    if (transform(x,n) == x)
```

Comment:
I'm worried that this test might fail due to roundoff errors.  Maybe it should apply to an `ivec` (integer coordinates) instead?

Reply:
Ok I will fix it

Reply:
I switched to ivec, and the code worked for me.